app/models/answer.rb

```diff
@@ -11,4 +13,9 @@ class Answer < ActiveRecord::Base
   def self.with_associations
     includes(:user).includes(:comments).with_votes
   end
+
+  def self.accept(id)
+    Answer.find(id).update_attribute("accepted", true)
```

Comment:
* I have removed the `Answer` :100: 
* Given that the upvote is clicked on an existing answer, I don't expect a case where the answer with the given `id` will not exist.

*Now that I have moved the concern to the `User` model, I think I can make it an instance method and treat your "case above :point_up: " :smile: